inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd

```diff
@@ -206,11 +206,37 @@ stock <- data %>%
 showLinePlots(stock, "Production|Industry|Steel|Accumulated Production", mainReg = "World")
```

Comment:
Sorry, I missed that in your last PR, but its better to not explicitly set `mainReg = "World"`. If you just remove this argument (here and in your calls to `showLinePlots()`) below, the function will use the default value for the main region. The default is `getOption("mip.mainReg")` which is globally set to the correct main region. This could be "World" but also e.g. "EUR".